ignite/contrib/handlers/param_scheduler.py

```diff
@@ -564,15 +569,22 @@ def save_history(self):
         return self._current_scheduler.save_history
 
     @save_history.setter
-    def save_history(self, value):
+    def save_history(self, value: bool):
         for s in self.schedulers:
             s.save_history = value
 
     def get_param(self):
         return self._current_scheduler.get_param()
 
     @classmethod
-    def simulate_values(cls, num_events, schedulers, durations, param_names=None, **kwargs):
+    def simulate_values(
+        cls,
+        num_events: int,
+        schedulers: List[ParamScheduler],
+        durations: List[int],
+        param_names: Union[List[str], Tuple[str]] = None,
```

Comment:
shouldn't it be `param_names: Optional[Union[List[str], Tuple[str]]] = None` here?

Reply:
Or even `Tuple[str, ...]` ?